recipes/sasview/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/SasView/sasview
+  git_rev: 390494d7cc6099973fce721a89afc666ae254c11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - numpy
+    - pyqt >=5
+    - python
+    - setuptools
+  run:
+    - bumps
+    - h5py >=2.5
+    - lxml
+    - matplotlib
+    - numpy
```

[See here](https://conda-forge.org/docs/meta.html#building-against-numpy):
```suggestion
    - {{ pin_compatible('numpy') }}
```